### ISSUE TITLE
ReCaptha and Formbuilder not showing/working

### ISSUE THREAD

**Issue by luca7766** [templaza/astroid-framework#898]
**Describe the bug**
Google ReCaptcha not showing/working in Form Builder element

**To Reproduce**
Testing the template (astroid one) in localhost environment, everything is almost good, except the recaptcha that I would like to use in a section where I insert a simple contact form using Form Builder integrated in the astroid framework+template.
I enabled the joomla integrated plugin for google recaptcha, then I enabled it in the form builder element, it won't show, so I made a simple contact form using standard contact component of joomla and it show just when I open the page (linked to a menu) of the joomla standard contact form, then sending with joomla standard contact form it correctly validate the recaptcha and send the email, but in the form builder generated form, captcha appears but always says "captcha is invalid".

In short:
Astroid Form builder's form won't show any recaptcha box unless a page with Joomla contact component is loaded, but the working captcha (and mail sent successfully) is just for the joomla standard contact module.

For recaptcha I'm using testing key as Google specified in this page:
https://developers.google.com/recaptcha/docs/faq?hl=en

I inspected the html code of the page and seems that the necessary js for google recaptcha aren't loaded (for the form builder element)

**Expected behavior**
I expected that ReCaptcha box show and work as in the Joomla classic contact form

**System Informations:**
 - Joomla Version: 4.4.9
 - PHP Version: 8.2.0
 - Astroid Version Framework Version: 3.1.8 - Template Version: 3.1.0
 - Device: ALL
 - OS: Windows 11
 - Browser: ALL

**Comment by sonvnn** [comment]
Thank you for report issue. I will check and recorrect it in the next version :)

**Comment by sonvnn** [comment]
I fixed this issue on https://github.com/templaza/astroid-framework/commit/a000628f9b273fa479c01bf94b6025f7a62d89db#diff-5c8a80c18a9271165685116a3654d2211be582b2ac8b9e26abe5928e93e85b38

You can download v3-branch and retest it.

Best Regards,
Sonny

**Comment by luca7766** [comment]
Thanks, it seems to be working now
